bigtable/hello/main.py

```diff
@@ -82,14 +84,18 @@ def main(project_id, instance_id, table_id):
     table.mutate_rows(rows)
     # [END writing_rows]
 
+    # [START creating_a_filter]
+    row_filter = row_filters.CellsColumnLimitFilter(1)
+    # [END creating_a_filter]
+
     # [START getting_a_row]
     print('Getting a single greeting by row key.')
     key = 'greeting0'.encode()
 
     # Only retrieve the most recent version of the cell.
```

Comment:
This comment applies to the "row_filter = ..." line

Reply:
done